lib/apiservers/engine/backends/container.go

```diff
@@ -1500,11 +1500,11 @@ func copyConfigOverrides(vc *viccontainer.VicContainer, config types.ContainerCr
 	vc.HostConfig = config.HostConfig
 }
 
-func externalIPv4Addrs() ([]netlink.Addr, error) {
-	l, err := netlink.LinkByName(externalIfaceName)
+func publicIPv4Addrs() ([]netlink.Addr, error) {
+	l, err := netlink.LinkByName(publicIfaceName)
 	if err != nil {
 		return nil, fmt.Errorf("Could not look up link from client interface name %s due to error %s",
```

Comment:
This message talks about client interface, not external - was this an error in the original code?